test/avocado/seleniumlib.py

```diff
@@ -81,10 +81,10 @@ def tearDown(self):
             screenshot_file = ""
             try:
                 # use time.clock() to ensure that snapshot files are unique and ordered
-                screenshot_file = "snapshot-teardown-%s.png" % str(time.clock())
+                screenshot_file = "screenshot-teardown-%s.png" % str(time.clock())
```

Comment:
why not save it to the right directory here instead of moving it later?
the same for the other cases where you save to a file